---
layout: single
permalink: /research/
title: Research
# toc: true
---


RNA viruses evolve rapidly and accumulate mutations as outbreaks grow and spread.
Because viral evolution is simultaneous with epidemiological growth, the evolutionary relationships among sequenced cases records important information about the processes that drive epidemics.
In the McCrone lab we use computational biology and phylogenetics to estimate these relationships and understand their implications for pathogen transmission.


<!-- 
I am interested in how processes that act on small scales interact with viral evolution and are manifest in global trends.
In my Ph.D.
thesis, I explored how within-host processes and transmission bottlenecks affect influenza evolution.
As a postdoc, I have worked to integrate epidemiological data such as human mobility with large phylodynamics models to explore the drivers of SARS-CoV-2 spread within the UK at a high resolution.
As an independent investigator, I will build on this experience to develop novel approaches for tractable, accessible phylodynamics at scale.
My research will explore how we can best apply large, genetic data sets to improve real-time public health responses and elucidate the mechanisms by which local dynamics affect large-scale virus evolution and epidemiology. -->




## High-resolution phylodynamics 
Historically, genetic data sets have lacked the density and resolution needed required for modeling pathogen spread in high detail.
However, the current SARS-CoV-2 pandemic has shown that we have entered a new era of genetic epidemiology.
Current and future data sets will be large enough to inform high-resolution models of viral epidemics.

{% include figure image_path="/assets/images/delta-example.png" alt="A phylogenetic tree of global delta spread and a map of spread within the UK" caption="Tracking Delta's import into and spread within England" %}

During the pandemic we were able to leverage  global and local sequencing efforts to reconstruct local dyanmics from genome sequences.
We are working to expand this work beyond pandemic response to learn about the fundamental processes that drive transmission in different viral systems.

## Novel methods for large-scale phylodynamics

Our ability to analyze tens of thousands of sequenced isolates greatly enhanced our understanding of SARS-CoV-2 evolution and spread.
Such analysis are feasible because of simple models and memory efficient data sctructures implemented in BEAST.

{% include figure image_path="/assets/images/thorney-work.png" alt="Two plots showing new methods are faster and use less memory" caption="Simple models and data structurse make large-scale analyses tractable" %}

We are incorporating these modeles into more sophisticated statistical frameworks to so that we can answer deeper biological questions about virus evoltion and make large-scale analyses rapid, routine, and accessible.

## Causes and consequences of within-host diversity


The global evolutionary dynamics of RNA viruses ultimately derive from processes that act within and between individual hosts.
But within-host datasets are noisy, and we have an incomplete understanding of how tranmission bottlenecks and within-host selective pressure shape viral diveristy.

{% include figure image_path="/assets/images/bottleneck.jpg" alt="A diagram of a genetic bottleneck and figure showing limitted transmitted diversity" caption="Transmission restricts the diversity as populations move between people" %}

In our group we are building new models and approaches to overcome these challenges and characterize the molecular evolution of viruses at the host level.


---
The recent and rapid growth in sequencing capacity has the potential to revolutionize our understanding of virus evolution and epidemiology.
We are working to realize that potential.
